.travis.yml

```diff
@@ -3,7 +3,7 @@ cache: bundler
 sudo: false
 branches:
   only:
-  - master
+  - atomic-tags
```

Comment:
This doesn't work, also never do this in a FOSS project.

Reply:
Absolutely, I was just running through travis for my local branch and forgot to revert it.